engine/storage/image/src/main/java/org/apache/cloudstack/storage/image/manager/ImageStoreProviderManagerImpl.java

```diff
@@ -157,6 +157,7 @@ public DataStore getImageStore(List<DataStore> imageStores) {
                 }
             }
         }
-        return imageStores.get(0);
+        s_logger.debug("Can't find staging storage in zone with less than 90% usage");
```

Comment:
isn't this threshold value meant to be configurable? change makes sense if not!
/cc @DagSonsteboSB @rhtyd

Reply:
The idea is that it shouldn't be.

Reply:
@kioie 90% cutoff value is defined here https://github.com/apache/cloudstack/blob/7e4a300a216a31348d2adbcc641d26e8339da205/server/src/main/java/com/cloud/server/StatsCollector.java#L290 and it can change in the future. Would it be better to not use constant in logs?